src/porepy/numerics/fv/fvutils.py

```diff
@@ -584,6 +584,34 @@ def map_hf_2_f(fno=None, subfno=None, nd=None, g=None):
     ).tocsr()
     return hf2f
 
+def sc_2_c(nd, sub_cell_index, cell_node_blocks, num_subhfno):
```

Comment:
This function does exactly the same as the function: map_sc_2_c ?
If so, can we merge them, e.g., include the length of the vector as dim:
 def map_sc_2_c(nd, sub_cell_index, cell_index, dim):?

Or call the functions sc_2_c_scalar and sc_2_c_vector?

Reply:
they do both mapping from subcells to cells, I'm pretty sure there is a way to merge them

Reply:
We can merge the two functions but we would not merge effectively nothing, that is we would still maintain the current code lines, and just add an if statement for vector/scalar problems.
These two functions do the same thing but on different problems, so also the ordering is different.
One (sc_2_c) maps a vector quantity (gravity) for a scalar problem (flow).
The other(map_sc_2_c) maps a scalar quantity (pressure) for a vector problem (mechanics).
I would rather change names (map_sc_2_c_vec_2_scal and map_sc_2_c_scal_2_vec) than implement another if statement.

Reply:
or map_sc_2_c_scalar and map_sc_2_c_vector as Runar suggested

Reply:
I understand. Then I lean towards two different functions. But their names should be updated.

If we follow the convention from mortar grids we should drop "map" and 2->to (even if I would argue that to-> 2 in mortar grid, but that is an other discussion :P ) 
Further, a scalar quantity is the default (e.g., boundary condition), so how about the names
cell_vector_to_subcell and cell_to_subcell_vector?

Reply:
they have dimensions of (num_subcells * nd, num_cells * nd) and (num_subcells * nd, num_cells) respectively, so I would call them cell_vector_to_subcell_vector and cell_scalar_to_subcell_vector.
We might also drop the last vector as they both map to a subcell vector (n_K in flow and alpha_grad in Biot).